### PR TITLE
Linux

### PR DESCRIPTION
**Pull request by msangapu-msft**
We're working with Michael Blythe on creating ARM quickstarts for docs.microsoft.com/azure/app-service. This is the Linux version. It's different than existing ARM templates in that, it has: a FREE SKU and flexibility to change programming languages easily in the template - which is a requirement for Quickstarts (day 1 scenarios). Please review and approve.

# Best Practice Checklist

Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment.
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by msangapu-msft** [comment]
Hi @bmoore-msft . This is failing for region usgovtexas. Does that need to be resolved before approval? This is not a gov-specific template.

**Comment by msangapu-msft** [comment]
Hi @bmoore-msft Please take a look when you have a moment.
We're working with Michael Blythe on creating ARM quickstarts for docs.microsoft.com/azure/app-service. This is the **Linux** version. It's different than existing ARM templates in that, it has: a FREE SKU and flexibility to change programming languages easily in the template - which is a requirement for Quickstarts (day 1 scenarios). Please review and approve.

**Comment by msangapu-msft** [comment]
Hi @bmoore-msft Please review.

File: 101-webapp-basic-linux/README.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
-# Deploy a app service plan and a basic Linux web app
+# This template allows you to deploy an App Service Plan and a Linux web app.
```
Comment (bmoore-msft) [inline 101-webapp-basic-linux/README.md:1]: fix header

File: 101-webapp-basic-linux/README.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-webapp-basic-linux/CredScanResult.svg)	
+![Best Practice Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-webapp-basic-linux/BestPracticeResult.svg)
+![Cred Scan Check](https://azurequickstartsservice.blob.core.windows.net/badges/101-webapp-basic-linux/CredScanResult.svg)
 
 [![Deploy To Azure](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazure.svg?sanitize=true)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-webapp-basic-linux%2Fazuredeploy.json)
-[![Visualize](https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/visualizebutton.svg?sanitize=true)](http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2F101-webapp-basic-linux%2Fazuredeploy.json)
-
-This template allows you to deploy an app service plan and a basic Linux web app.
```
Comment (bmoore-msft) [inline 101-webapp-basic-linux/README.md:15]: need a description

File: 101-webapp-basic-linux/azuredeploy.json [hunk 3]
```diff
@@ -1,70 +1,68 @@
...
+            "properties":{
+             "reserved":true
+            }
+        },
+        {
+            "apiVersion": "2019-08-01",
+            "type": "Microsoft.Web/sites",
+            "kind": "app",
+            "name": "[parameters('webAppName')]",
+            "location": "[resourceGroup().location]",
```
Comment (bmoore-msft) [inline 101-webapp-basic-linux/azuredeploy.json:56]: must be parameters('location')

File: 101-webapp-basic-linux/azuredeploy.json [hunk 4]
```diff
@@ -1,70 +1,68 @@
...
+            "properties": {
+                "siteConfig": {      
+                    "linuxFxVersion": "[parameters('linuxFxVersion')]"
+                },
+                "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', variables('appServicePlanPortalName'))]"
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 101-webapp-basic-linux/azuredeploy.json:63]: move this above properties

File: 101-webapp-basic-linux/azuredeploy.parameters.json [hunk 5]
```diff
@@ -4,6 +4,6 @@
 	"parameters": {	
 		"webAppName": {
 			"value": "GEN-UNIQUE"
-		}
+		  },
```
Comment (bmoore-msft) [inline 101-webapp-basic-linux/azuredeploy.parameters.json:7]: remove extraneous command and format the JSON

**Comment by bmoore-msft** [comment]
Closing abandoned request - please reopen if you can address the feedback.

**Pull request closed without merging**